R/deployApp.R

```diff
@@ -114,7 +114,9 @@ deployApp <- function(appDir = getwd(),
                       forceUpdate = getOption("rsconnect.force.update.apps", FALSE),
                       python = NULL,
                       on.failure = NULL,
-                      forceGeneratePythonEnvironment = FALSE) {
+                      forceGeneratePythonEnvironment = FALSE,
+                      shinyAppsVisibility = "public"
```

Comment:
Seems a bit odd to include `shinyapps` in the parameter name. What about `appVisibility`? Obviously, this is a shinyapps.io specific flag, but we've never explicitly called out RSC specific flags.